File: LeetCode/powerOfTwo.py
class Solution(object):
    def isPowerOfTwo(self, n):
        """
        :type n: int
        :rtype: bool
        """
        if n <= 0:
            return False
            
        oneBitCount = 0
        for i in range(32):
            if ( (n & 1) == 1):
                oneBitCount += 1
                
            n = n >> 1
        return oneBitCount == 1

File: LeetCode/powerOfTwo.cpp
class Solution {
public:
    bool isPowerOfTwo(int n) {
        
        if(n <= 0){
            return false;
        
    }
    
    int oneBitCount = 0;
    for(int i = 0; i < 32; i++){
        if(n & 1 == 1){
            oneBitCount++;
        }
        
    n = n >> 1;
    
    }
    
    return oneBitCount == 1;
    }
};